js/main.js

```diff
@@ -3,20 +3,27 @@
 
 Site = {
   mobileThreshold: 601,
+  $body: $('body'),
+  $mailchimpForm: $('#mailchimp-form'),
```

Comment:
The conditional check can just go in the init function of Mailchimp object and if you want to be mega clean you could but the bind in a bind function as that is kinda a pattern me and cas have been using